pyproject.toml

```
@@ -5,8 +5,8 @@ build-backend = "flit_core.buildapi"
 [project]
 name = "xsdba"
 authors = [
-  {name = "Éric Dupuis", email = "dupuis.eric@ouranos.ca"},
-  {name = "Trevor James Smith", email = "smith.trevorj@ouranos.ca"}
+  {name = "Pascal Bourgault", email = "bourgault.pascal@ouranos.ca"},
+  {name = "Éric Dupuis", email = "dupuis.eric@ouranos.ca"}
```

```suggestion
  {name = "Éric Dupuis", email = "dupuis.eric@ouranos.ca"},
  {name = "Pascal Bourgault", email = "bourgault.pascal@ouranos.ca"}
```
I think the first author should be the main point of contact!